packages/e2e-tests/fixtures/blocks/core__buttons.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:buttons -->
-<div class="wp-block-buttons">
+<div class="wp-block-buttons is-direction-horizontal">
```

Comment:
Adding a vertical version of these test fixtures would be great. Let me know if you can't recall how to do that and I can write up some instructions (I should really improve the README file!).

Reply:
Okay, I [tried to update the tests](https://github.com/WordPress/gutenberg/pull/20160/commits/fd14a30995d402dcd1c97a30ca0060cb34b1bca5) but I think I did it wrong. The [Javascript unit test](https://travis-ci.com/github/WordPress/gutenberg/jobs/318407633) failed.

Could you possibly review my updates once and see what I did wrong?

Thank you!